brand/models/brand.py

```diff
@@ -22,54 +22,63 @@ class Brand(TimeStampedModel):
     """
 
     name = models.CharField(
-        "Name of this brand/data source",
-        max_length=200,
-        null=False,
-        blank=False,
-        default="-unnamed-",
+        "Name of this brand", max_length=200, null=False, blank=False, default="-unnamed-"
     )
+    name_locked = models.BooleanField(default=False)
     description = models.TextField(
-        "Description of this instance of this brand/data source",
-        null=True,
-        blank=True,
-        default="-blank-",
+        "Description of this instance of this brand", null=True, blank=True, default="-blank-"
     )
+    description_locked = models.BooleanField(default=False)
     website = models.URLField(
-        "Website of this brand/data source. i.e. bankofamerica.com", null=True, blank=True
+        "Website of this brand. i.e. bankofamerica.com", null=True, blank=True
     )
+    website_locked = models.BooleanField(default=False)
     countries = CountryField(
         multiple=True, help_text="Where the brand offers retails services", blank=True
     )
+    countries_locked = models.BooleanField(default=False)
```

Comment:
The Countries field is a special case. It should not be lockable because merging countries is additive.

Reply:
done